library/style-guide.css

```diff
@@ -0,0 +1,18 @@
+/* flex and column utility are used in alignment in library */
+@import url("../utils/utility.css");
+
+:root{
+    --RED :  #FF715B;
```

Comment:
try to give color names based on the functionality rather than actual color name